lib/board.rb

```diff
@@ -0,0 +1,74 @@
+class Board
+
+  WIN_COMBINATIONS = [[0, 1, 2], 
+                      [3, 4, 5],
+                      [6, 7, 8],
+                      [0, 3, 6],
+                      [1, 4, 7],
+                      [2, 5, 8],
+                      [0, 4, 8],
+                      [2, 4, 6]]
+
+  attr_accessor :cells
+
+  def initialize
+    @cells = [" ", " ", " ", " ", " ", " ", " ", " ", " "]
```

Comment:
Looks like you already have this code elsewhere in your class. Here you could just call the `reset!` method.